index.html

```diff
@@ -0,0 +1,112 @@
+                <li class="dt_nav_bar_item"><img class="dt_facebook-link" alt="facebook" src="./Assets/Images/Icons/facebook.png"></li>
+                <li class="dt_nav_bar_item"><img class="dt_twitter-link" alt="twitter" src="./Assets/Images/Icons/twitter.png"></li>
+                <li class="dt_nav_bar_item"><a class="dt_english-link">English</a></li>
+                <li class="dt_nav_bar_item"><a class="dt_my-page-link">My Page</a></li>
+                <li class="dt_nav_bar_item_last"><a class="dt_logout-link">Logout</a></li>
+            </ul>
+            <ul class="dt_navbarbottom">
+                <li class="dt_bottom_bar_item_first"><a href="./index.html"><img class="item_first_img" alt="home" src="./Assets/Images/Logos/ifclogo.png"></a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="./about.html">About</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_2">Program</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_3">Speakers</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_4">Sponsors</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_5">Credits</a></li>
+                <li class="dt_bottom_bar_item_last"><a class="dt_cc-campaign-link">INT Food C.</a></li>
+            </ul>
+    </nav>
+    <div id="menu_display" class="menu_display">
+        <img id="close_menu_btn" class="close_menu_btn" alt="close" src="./Assets/Images/Icons/close.png">
+        <ul class="md_links">
+            <li class="md-item"><a class="md-link-asside" href="./about.html">About</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_2">Main Program</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_3">Featured Speakers</a></li>
+        </ul>
+    </div>
+    <section id="sec_1" class="sec_1">
+        <h1 class="sec_1_greetings">"Hello! Chefs community"</h1>
+        <h2 class="sec_1_title">INTERNATIONAL FOOD CONFERENCE</h2>
+        <h2 class="sec_1_title2">2021</h2>
+        <p class="sec_1_parraph">The most important meeting is here, learn from the greatest, best chefs around the world explain their secrets to rise your professional chef skills.</p>
+        <h2 class="sec_1_date">2021.07.28(WED) ~ 30(FRI)</h2>
+        <h3 class="sec_1_location">@Exposition park, Sheraton conference room and more</h3>
+    </section>
+    <section id="sec_2"  class="sec_2">
+        <h1 class="sec_2_title">Main Program</h1>
+        <div class="line_separator"></div>
+        <ul class="program_boxes">
+            <li class="pr_box">
+                <img class="pr_box_icon" alt="Lecture" src="./Assets/Images/Icons/lecture.png">
+                <h1 class="pr_box_title">Lecture</h1>
+                <p class="pr_box_parraph">Listen to chefs around the world about their techniques and the way they walkthrough kitchen issues.</p>
```

Comment:
* These are articles and should be wrapped in `article` tags, please make the correction in other places it reoccurs